_posts/2018-04-11-dont-make-users-think-about-updating-software.md

```diff
@@ -11,35 +11,35 @@ tags:
 
-Take a step back to look at the browser you are using to view the web. Do you know what version it is? Do you care? _Should you care_? **No**. There was a time when you would only get an update to your browser when you updated your operating system. There was a time when you needed to go to the vendor website to download a newer version. This caused severe damage the web as the web evolved; Certain features were only available in certain versions of browsers. This is still an issue today.
+Take a step back to look at the browser you are using to view the web. Do you know what version it is? Do you care? _Should you care_? **No**. There was a time when you would only get an update to your browser when you updated your operating system. There was a time when you needed to go to the vendor's website to download a newer version. This caused severe damage the web as the web evolved; certain features were only available in certain versions of browsers. This is still an issue today.
 
-> Going to a website to download an installer to keep someone else's software current should not be a responsibility of the user
+> Going to a website to download an installer to keep someone else's software current should not be a responsibility of the user.
 
-Modern browsers and app stores remove the hurdles to keeping your and their software evergreen. The term `evergreen` might have become popular when Google Chrome started auto updating itself in late 2010. Since then, all major browsers have switched to auto updating. Browser auto updating works by when you start your browser, it checks for an update and downloads it in the background. The next time you start the software, you are using the newest version. I think this set a precedent for software that users expect and desire. Going to a website to download an installer to keep someone else's software current should not be a responsibility of the user. It is a great way to turn away users.
+Modern browsers and app stores remove the hurdles to keeping your and their software evergreen. The term `evergreen` might have become popular when Google Chrome started auto updating in late 2010. Since then, all major browsers have switched to auto updating. Browser auto updating is pretty straightforward&mdash; when you start your browser, it checks for an update and downloads it in the background. Then, the next time you start the software, you are using the newest version. I think this set a precedent for software that users expect and desire. Going to a website to download an installer to keep someone else's software current should not be a responsibility of the user. And placing this responsibility on the users is a great way to turn them away.
```

Comment:
Changed that "Browser auto updating is pretty straightforward" for readability again. The "Browser auto updating works by when..." wording threw me off a little bit. 

Also, auto updating: Chicago Manual of Style (which I typically follow for AGRC stuff) usually recommends making these kinds of terms one word, no hyphen. Consistency is most important, but personally, I'd recommend either adding a hyphen or making it one word no hyphen.

Reply:
I'll switch it to one word

Reply:
![image](https://user-images.githubusercontent.com/325813/38701830-29a33560-3e5c-11e8-8f54-50e377718135.png)

even though a hyphen is twice as popular